cmd/rookflex/cmd/mount.go

```diff
@@ -235,9 +236,16 @@ func mountCephFS(client *rpc.Client, opts *flexvolume.AttachOptions) error {
 		}
 	}
 
+	// write secret key to temporary file
+	secretfile, err := util.CreateTempFile(clientAccessInfo.SecretKey)
+	if err != nil {
+		return fmt.Errorf("failed to create temporary file: %+v", err)
```

Comment:
```suggestion
		return errors.Wrapf(err, "failed to create temporary file")
```
Please use the new `"github.com/pkg/errors"` package.